HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift

```diff
@@ -111,8 +111,7 @@ public final class PaymentReviewViewController: UIViewController, UIGestureRecog
                 if let paymentProviders = self?.paymentProviders, paymentProviders.count > 0 {
                     let providerId = UserDefaults.standard.string(forKey: "ginihealth.defaultPaymentProviderId")
                     let provider = paymentProviders.first(where: { $0.id == providerId }) ?? paymentProviders[0]
-                    self?.configureBankProviderView(paymentProvider: provider)
-                    self?.configurePayButton(paymentProvider: provider)
```

Comment:
These two removed method calls look important. Are you sure they are not needed anymore?

Reply:
These lines will be called automatically when `selectedPaymentProvider` will be set                     `self?.selectedPaymentProvider = provider`

Reply:
Thanks, I understand the change now.